package-overrides/github/gaearon/react-transform-catch-errors@1.0.0.json

```diff
@@ -0,0 +1,5 @@
+{
+  "registry": "npm",
+  "main": "src/index",
+  "format": "es6"
```

Comment:
The only way I could prevent the double interop from breaking everything was to use the original ES6 sources.  As for not using `npm:react-transform-catch-errors`, that was because @gaearon removes the `src` directory when publishing to npm, which is of course needed for the `ES6` code.